code/dff.py

```diff
@@ -1,63 +1,82 @@
 import argparse
 import json
+import logging
 import os
 from datetime import datetime as dt
 from pathlib import Path
 from typing import Union
 
 import aind_ophys_utils.dff as dff
 import h5py
-import numpy as np
-from aind_data_schema.core.processing import (DataProcess, PipelineProcess,
-                                              Processing, ProcessName)
+from aind_data_schema.core.processing import DataProcess, ProcessName
+from aind_log_utils.log import setup_logging
 from scipy.stats import skew
 
 
-def write_output_metadata(
+def write_data_process(
     metadata: dict,
-    process_json_dir: str,
-    process_name: str,
     input_fp: Union[str, Path],
     output_fp: Union[str, Path],
-    start_date_time: dt,
+    unique_id: str,
+    start_time: dt,
+    end_time: dt,
 ) -> None:
     """Writes output metadata to plane processing.json
 
     Parameters
     ----------
     metadata: dict
         parameters from suite2p motion correction
-    input_fp: str
-        path to data input
+    raw_movie: str
```

Comment:
argument name is input_fp, not raw_movie. And the input is not a raw movie, but (neuropil-corrected) traces